### DISCUSSION TITLE
Is production ready with pgbouncer + postgres?

### DISCUSSION THREAD

**Discussion by budibong** [aarondl/sqlboiler#1216] [Q&A] [answered]
Hi everyone,

I am thinking to adopt sqlboiler into my team tech stack, but i still have some questions.

1. Does anyone have succesfully using sqlboiler for production application  with high volume of transactions ? Especially, using pgbouncer + postgres

2. For postgres, is the generated code already use pgx as the driver ?

Any help n advice will be very appreciated. Thanks

**Comment by stephenafamo** [comment, marked as answer]
Because [`pgx.Conn`](https://pkg.go.dev/github.com/jackc/pgx/v5#Conn) has query and exec methods that don't return `*sql.Rows` (they return `*pgx.Rows`) then it does not satisfy the interface that the generated SQLBoiler code expects to query with.

You can use the stdlib adapter from pgx (<github.com/jackc/pgx/v5/stdlib>).

**Reply by stephenafamo** [reply]
For pgbouncer, since those work like any postgres connection, they should work fine with SQLBoiler
